applescript/functions

```diff
@@ -10,40 +27,31 @@ function is-terminal-active() {
     zstyle -s ':notify:' plugin-dir plugin_dir
     source "$plugin_dir"/lib
 
-    # run an AppleScript, selected from ./resources by basename given as the
-    # first argument, with all other arguments are positional arguments to the
-    # script's `on run` handler.
-    function run-applescript() {
-        local script_name
-
-        script_name="$1"
-        shift
-
-        "$plugin_dir"/applescript/resources/"$script_name".applescript $@ 2>/dev/null
-    }
-
     # exit with code 0 if the terminal window/tab is active, code 1 if inactive.
     function is-terminal-window-active {
-        local term
+        local script_name script_arg
 
         if [[ "$TERM_PROGRAM" == 'iTerm.app' ]] || [[ -n "$ITERM_SESSION_ID" ]]; then
-            term=iterm2
+            script_name=is-iterm2-active
+            script_arg=$(current-tty)
         elif [[ "$TERM_PROGRAM" == 'Apple_Terminal' ]] || [[ -n "$TERM_SESSION_ID" ]]; then
-            term=apple-terminal
+            script_name=is-apple-terminal-active
+            script_arg=$(current-tty)
         else
-            return 2
+            script_name=is-window-active-by-pid
+            script_arg=$(top-level-ppid)
         fi
 
-        run-applescript is-"$term"-active "$(current-tty)"
+        run-applescript $script_name $script_arg
     }
 
     if is-terminal-window-active; then
         if is-inside-tmux; then
             is-current-tmux-pane-active
             return $?
         fi
-      else
-          return $?
+        else
+            return $?
```

Comment:
I think this is less understandable then it was before — this clause is part of the outer `if`, so it should either be the way it was before, or on the same level (4 spaces less indent) as the outer `if`.

Reply:
Yes, my bad. Fixed it